docs/developers/quickstart/proof-composition.md

```diff
@@ -2,9 +2,177 @@
 
 Hylé's native proof verification allows for proof composition. To understand the concept better, we recommend you [read this blog post](https://blog.hyle.eu/proof-composability-on-hyle/).
 
-This guide walks you through creating and deploying your first ticket transfer contract, based on a ticket-app and a simple-token app, by using Hylé and RISC Zero. 
+This guide walks you through creating and deploying your first ticket transfer contract, based on a ticket-app and a [simple-token app](./your-first-smart-contract.md), by using Hylé and RISC Zero.
 
-Find the source code here:
+Find the source code for both contracts here:
 
 - [ticket-app](https://github.com/Hyle-org/examples/tree/feat/ticket-app/ticket-app)
 - [simple-token](https://github.com/Hyle-org/examples/tree/feat/ticket-app/simple-token)
+
+## Quickstart
+
+### Prerequisites
+
+- [Install Rust](https://www.rust-lang.org/tools/install) (you'll need `rustup` and Cargo).
+- [Install OpenSSL crate](https://crates.io/crates/openssl).
+- For our example, [install RISC Zero](https://dev.risczero.com/api/zkvm/install).
+- [Start a single-node devnet](./devnet.md). We recommend using [dev-mode](https://dev.risczero.com/api/generating-proofs/dev-mode) with `-e RISC0_DEV_MODE=1` for faster iterations during development.
+
+### Simple-token preparation
```

Comment:
Maybe before this a sentence to say the workflow we are going to follow:

- Register an identity contract
- Register 2 identites
- Register a token contract
- Faucet the users with this token
- Register the ticket-app contract
- Buy a ticket

Where each line has a link to the corresponding section